### ISSUE TITLE
2.5.0 un-usable - users_clothing errors

### ISSUE THREAD

**Issue by brenoepics** [brenoepics/morningstar#885]
In GitLab by @ghost on Oct 29, 2020, 17:34

Ok so easily put this is why people with custom clothing sellable (such as me or Habpop had) can't upgrade to the next build (specifically 2.5.0 or @beny's subscription revamp.

## What happens?
The linking is expected to work likewise:
```
SELECT users_clothing.*, catalog_clothing.setid FROM users_clothing LEFT JOIN catalog_clothing ON catalog_clothing.id = users_clothing.clothing_id WHERE users_clothing.user_id = ?
```
So `users_clothing` contains the `catalog_clothing.clothing_id` which is tracked back to `catalog_clothing.setid`. This however does not work on random cases. I have no idea why, but it makes people unable to connect to the actual Hotel, generating a bunch of random errors specific for random users. This is what is specific to a random user (**I'm also unable to connect**):
https://i.imgur.com/XvNpDQE.png - this would somewhat explain it but I don't get why it would split it likewise https://cdn.discordapp.com/attachments/305770861577764884/766785888951730196/unknown.png

**What's fascinating is why can't me or another user connect, because honestly I don't have any reasons for it (besides the random NULL):**
me: https://i.imgur.com/BTSK6A9.png
other user: https://i.imgur.com/eaPX8oj.png

See anything wrong with the other users query? yeah me neither, but he still can't connect, probably due to the actual custom clothing. Reclaiming these clothing did not cause any errors on another user, but for an odd reason it did for him. This makes me believe the error might be that he claimed a clothing that he already had, which would be `clothing_pjbottoms1` (**147**). 

## LONG STORY SHORT
I believe the error is actually that people wear clothing that exist in `catalog_clothing`, they claimed it and its **sellable="0"**, or they have users_clothing that's null but I have no idea why they exist or how they appeared. That's easy to truncate, but I'm fairly certain it will happen again.

This is a really confusing error so sorry about that.
